exercises/two-fer/README.md

```diff
@@ -0,0 +1,40 @@
+`Two-fer` or `2-fer` is short for two for one. One for you and one for me.
+
+```text
+"One for X, one for me."
+```
+
+When X is a name or "you".
+
+If the given name is "Alice", the result should be "One for Alice, one for me."
+If no name is given, the result should be "One for you, one for me."
+
+## Test-Driven Development
```

Comment:
Please remove this section from the README.

That means from here up to but not including the line `## Instructions`